staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go

```diff
@@ -543,7 +543,16 @@ func (c *AvailableConditionController) updateAPIService(oldObj, newObj interface
 	if !reflect.DeepEqual(castObj.Spec.Service, oldCastObj.Spec.Service) {
 		c.rebuildAPIServiceCache()
 	}
-	c.queue.Add(oldCastObj.Name)
+	c.checkIfNotInBackoff(oldCastObj.Name)
+}
+
+func (c *AvailableConditionController) checkIfNotInBackoff(apiservice string) {
```

Comment:
I am happy the issue could be solved with the small patch, but let's choose another name for the method? 
_checkIfNotInBackoff_ means the method checks only, while it Is actually adds apiservice to the queue.

I'd suggest something like `queueAddIfNotInBackoff` or `enqueueIfNotRequeued`

Reply:
Hi, agreed - changed to queueAddIfNotInBackoff